docs/guides/security/backups/create-restic-repository-shortguide/index.md

```diff
@@ -12,26 +12,30 @@ title: "Install Restic backup"
 headless: true
 show_on_rss_feed: false
 ---
+
+{{< note >}}
 [Create an Object Storage access key pair](/docs/platform/object-storage/how-to-use-object-storage/#generate-a-key-pair) if you have not done so already.
+{{< /note >}}
 
-Run the below command, replacing "your-key" with your access key, "your-secret" with your key's secret and "your-bucket-name" with the name of your bucket:
+1. Run the below command, replacing `your-key` with your access key, `your-secret` with your key's secret, and `your-bucket-name` with the name of your bucket:
 
-    AWS_ACCESS_KEY_ID=your-key AWS_SECRET_ACCESS_KEY=your-secret restic -r s3:us-east-1.linodeobjects.com/your-bucket-name init
+        AWS_ACCESS_KEY_ID=your-key AWS_SECRET_ACCESS_KEY=your-secret restic -r s3:us-east-1.linodeobjects.com/your-bucket-name init
+
+    {{< note >}}
+The above command references the `us-east-1` cluster, which is located in the Newark, NJ region. If your bucket is located in a different cluster, replace `us-east-1` with the appropriate cluster name.
 
-{{< note >}}
-If your bucket is not in the Newark, NJ region, replace "us-east-1.linodeobjects.com" with the cluster name where your bucket is located.
 Example: for Frankfurt, DE, the command would be:
 
     AWS_ACCESS_KEY_ID=your-key AWS_SECRET_ACCESS_KEY=your-secret restic -r s3:eu-central-1.linodeobjects.com/your-bucket-name init
 {{< /note >}}
 
-{{< note >}}
+    {{< caution >}}
```

Comment:
I thought a caution tag could work here, because a syntax error could leads to a different 'billable' service than the user expected (well, kind of..)